tests/codegen/generateViewFunctionsSugar.test.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from "bun:test";
+import path from "path";
+import fs from "fs";
+import os from "os";
+import { generateSugar } from "../../src/codegen/parseViewFunctions";
+
+const FIXTURES_DIR = path.join(__dirname, "../fixtures");
+
+test("generateSugar() writes a valid TypeScript file with sugar functions that compiles", () => {
+  // Create a temp directory
+  const tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), "viewgen-sugar-test"));
+  const outFile = path.join(tmpDir, "viewFunctionsSugar.ts");
+
+  // Calculate relative import path from generated file to src/types/clientPayloads
+  const relImport = path
```

Comment:
Suggest to the AI intern that this function appears twice?

Reply:
Looks like the same function in the other file.

Reply:
all moose are nearsighted